build.yaml

```diff
@@ -69,6 +71,40 @@ release:
             ./build_${PACKAGE_TYPE}.sh
             find build -type f -name "*.${PACKAGE_TYPE}" -exec mv {} packages \;
             pushd packages
+
+            # Install cpp-driver packages
+            sudo ${PACKAGE_INSTALL} cassandra*.${PACKAGE_TYPE}
+
+            # Create test file against cpp-driver and try to connect. Should compile and throw a connection error when executed
+            $CC -x c -o $TEST_PROGRAM - -lcassandra - <<EOF
```

Comment:
You can remove the includes if you add `-Wno-implicit-function-declaration` to your compile command.

Reply:
e.g. `echo 'int main() { cass_cluster_new(); printf("Hi\n"); }' | clang -x c - -Wno-implicit-function-declaration -lcassandra`

Reply:
Added